config/locales/models/courses/en.yml

```diff
@@ -5,6 +5,7 @@ en:
       course:
         display_name: Display Name
         is_hidden: Course Visibility
+        max_file_size: Maximum File Size (bytes)
```

Comment:
It would be good to clarify that this is for submissions, e.g. "Maximum Submission File Size". Or shorter, "Maximum Submission Size". Since there are other types of files that can be uploaded for a course.

Reply:
That is true, but we are also limiting the size of those files. Both autotest files and starter files cannot be larger than this value.